docs/sources/tutorials/get-started.md

```diff
@@ -1,7 +1,7 @@
 ---
 canonical: https://grafana.com/docs/alloy/latest/tutorials/get-started/
-description: Getting started with the tutorials
-title: Get started
+description: Getting started with Alloy
+title: Get started with Alloy
 weight: 10
 ---
 
```

Comment:
Under the ##Get Started with Alloy section, I propose that we replace this entire section with the following to give the readers an overview of what exactly they will be accomplishing after following this tutorial. 

This tutorial will walk you through configuring Alloy to collect logs from your local machine and report logs to Loki running in the local Grafana Stack. This process will enable you to query and visualize the logs sent to Loki using the Grafana dashboard. 

To follow this tutorial, you must have a basic understanding of [Alloy](https://grafana.com/docs/alloy/latest/) and telemetry collection in general. You should also be familiar with Prometheus and PromQL, Loki and LogQL, and basic Grafana navigation. You don’t need to know about the Alloy configuration syntax concepts.